native/fs-hdfs/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "datafusion-comet-fs-hdfs3"
+version = "0.1.12"
+edition = "2021"
+
+description = "libhdfs binding library and safe Rust APIs"
+authors = ["Yanghong Zhong <nju_yaho@apache.org>"]
+license = "Apache-2.0"
+
+keywords = ["hdfs", "hadoop"]
+documentation = "https://docs.rs/crate/fs-hdfs3"
+homepage = "https://github.com/datafusion-contrib/fs-hdfs"
+repository = "https://github.com/datafusion-contrib/fs-hdfs"
```

Comment:
We won't be publishing this crate (as far as I know), so we can either remove these entries or update them to point to the Comet repo.

We should also add `publish = false` to prevent accidenty publishing the crate to crates.io